crates/uv-git/src/git.rs

```diff
@@ -478,486 +399,53 @@ impl<'a> GitCheckout<'a> {
     ///
     /// [`.cargo-ok`]: CHECKOUT_READY_LOCK
     fn reset(&self) -> Result<()> {
-        let ok_file = self.path.join(CHECKOUT_READY_LOCK);
+        let ok_file = self.repo.path.join(CHECKOUT_READY_LOCK);
         let _ = paths::remove_file(&ok_file);
-        debug!("reset {} to {}", self.repo.path().display(), self.revision);
+        debug!("reset {} to {}", self.repo.path.display(), self.revision);
 
+        // TODO: see if this still necessary
         // Ensure libgit2 won't mess with newlines when we vendor.
-        if let Ok(mut git_config) = self.repo.config() {
-            git_config.set_bool("core.autocrlf", false)?;
-        }
+        // if let Ok(mut git_config) = self.repo.config() {
+        //     git_config.set_bool("core.autocrlf", false)?;
```

Comment:
Also unsure if we need to do this.

Reply:
I think we can just remove this. We don't support vendoring right? Unless I'm misunderstanding.

Reply:
What sort of vendoring are you referring to?

Reply:
Ah you mean cargo vendoring. Yeah that looks to be why they needed this, it shouldn't affect us. I'll remove it.